### ISSUE TITLE
[Feature Request] Renamable Loot - Player Loot drop list.

### ISSUE THREAD

**Issue by Sourceshard** [GoldenKittenPlays/GetLootForKills#8]
I have been adding plushies to the loot pools of monsters with this mod for my mod pack Shattered Company. I stumbled across a sinister idea when I saw a Scavenger Plushie.

I would love the player to have a 100% chance to drop a plushie with their player name and possibly cause of death on it. I am sure this is not in the scope of your mod. However, the idea excites me, I would love it and had to ask. I think it would be hilarious to see a pool of blood and a plushie sitting there as an erie reminder of those who have past.

A landmark for Laviathin deaths. Locations of where Brackens grabbed their victims. Where players were turned into a mimic. haha

https://thunderstore.io/c/lethal-company/p/Scintesto/Scavenger_Plushie/

**Comment by GoldenKittenPlays** [comment]
Yup probably would make it a separate mod but it is a cool idea and when I have an idea I might hop on to it. Thanks for the cool idea
